routes/index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const router = express.Router();
+const numberGuesser = require('../public/javascripts/NumberGuesser')
+const ScrabbleBag = require('../public/javascripts/ScrabbleBag')
+const FibonacciBase = require('../public/javascripts/FibonacciBase')
+const StringIndices = require('../public/javascripts/StringIndices')
+const disemvoweling = require('../public/javascripts/disemvoweling')
+
+/* GET home page. */
+router.get('/', function(request, response, next) {
+  const { guess, exit } = request.query
+  if(guess === 'exit' || exit === 'exit') {
```

Comment:
you're checking `guess === 'exit'` here and within `numberGuesser`. I think it's better to handle that here but def not in both places